### ISSUE TITLE
Implement visibility :root selector for amp-inabox

### ISSUE THREAD

**Issue by lannka** [ampproject/amphtml#7830]
This is a task split off from #7457, to unblock ActiveView in amp-inabox

/cc @dvoytenko

**Comment by dvoytenko** [comment]
@lannka #7457 is complete. However, I don't know how to confirm/validate that in-a-box case is fulfilled. In particular, it's important to consider the following issues: #7992 (InOb polyfill bugs) and #7971 (InOb vs viewport offsetting).

**Comment by lannka** [comment]
#7992 is fixed. 
we discussed about #7971, we will revisit once we have our LayerManager infrastructure in place.
I'll verify if this is fixed for inabox, and update this thread.

**Comment by lannka** [comment]
To confirm, `:root` selector works in inabox now. I improved the example for a thorough test #8180.
Found a minor issue: there is some URL vars missing, such as `elementX`, `elementY` will file a separate issue to track.
